doc/_toc.yml

```diff
@@ -29,6 +29,7 @@ parts:
       - file: apps/panel
       - file: apps/chainlit
       - file: apps/gradio
+      - file: apps/shiny-core
```

Comment:
let's call this shiny, I'm unfamiliar with the "shiny core" term and other might be confused as well

Reply:
[fixed](https://github.com/ploomber/doc/blob/6318fe9c144499deef0d0550c0626f73f6942aaf/doc/_toc.yml#L32)